### PR TITLE
task1

### PR DESCRIPTION
**Pull request by volodya0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
-# csdt2223ki48yuziukvs28
+Student: Volodymyr Yuziuk
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md:1]: do not use "." in tag name

**Review by andrii-s-shpitser-lpnu-ua — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (volodya0) [reply README.md:1]: Tag name renamed.
I'm not sure if renaming is enough, maybe I need to create a new release, please let me know

Comment (andrii-s-shpitser-lpnu-ua) [reply README.md:1]: image

Comment (andrii-s-shpitser-lpnu-ua) [reply README.md:1]: I still see dot

Comment (volodya0) [reply README.md:1]: Yes, strange, it loks like i can not rename the tag completely, now i see two different names (old and new) in different places. 
I will create new release

**Review by andrii-s-shpitser-lpnu-ua — APPROVED** [review]

**Pull request merged**